### DISCUSSION TITLE
Windows 10 21H2 and Windows 10 22H2 Taskbar

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2971] [Ideas]
is it possible to implement the taskbar from the version 22H2? Because I am pretty sure Explorer Patcher has the Windows 10 21H2 taskbar.
Here's the difference

Windows 10 21H2 
![IMG_1333](https://github.com/valinet/ExplorerPatcher/assets/152800265/99e1332e-5f50-44b3-b352-4b4357723a4e)
the search box is brighter (same thing in Explorer Patcher)
The Windows icon is a bit different from 22H2 

Windows 10 22H2 
![IMG_1334](https://github.com/valinet/ExplorerPatcher/assets/152800265/3c34706f-7621-47de-bbde-7add4f8fc8fe)
In 22H2 the search box is a lot darker, and I feel like the Windows icons were also changed. The angle of the Windows icons are both a bit different. Both 21H2's and 22H2's search BUTTON's icons are also different. 21H2's search button/icon is a tiny magnifying glass, in 22H2's their search button's/icon's magnifying glass is bigger. (I forgot to send the pictures of the search button/icon and cannot add the pictures of the search button/icon after I made this discussion)
I feel like it would be a nice feature to include both the 21H2 and 22H2 taskbar in Explorer Patcher.

**Comment by haydengriz** [comment]
I think the big search icon is getting added but the search box is a Very tiny detail

**Reply by TimMarvelHero** [reply]
but the big search icon is the tiny detail here. Not the search box . And the search box was added, just the one from 21H2, not from 22H2. In this discussion I've mentioned about potentially adding the taskbar from 22H2.

**Reply by haydengriz** [reply]
what's the difference between the two exactly?

**Reply by TimMarvelHero** [reply]
can you specify?

**Reply by TimMarvelHero** [reply]
the difference between the pictures?

**Comment by Amrsatrio** [comment]
Actually, the search box properties are controlled by Microsoft's servers. In Windows 11 22H2+, the mechanism for pulling the data is no longer present. But sure I can change the colors for the search box in the reimplemented taskbar, however I'm not sure about the width as it varies by language.

**Reply by TimMarvelHero** [reply]
Okay, will wait for the update.
